src/Foundation/Connect/code/Wooli.Foundation.Connect.csproj

```diff
@@ -588,6 +588,8 @@
     <Compile Include="Services\Search\ISearchService.cs" />
     <Compile Include="Managers\Pricing\IPricingManagerV2.cs" />
     <Compile Include="Managers\Pricing\PricingManagerV2.cs" />
+    <Compile Include="Managers\IOrderManagerV2.cs" />
+    <Compile Include="Managers\Search\ISearchManagerV2.cs" />

```

Comment:
Couldn't find ISearchManagerV2.cs in MR

Reply:
It renamed to `ISearchService`, I don't know why it still here after rebase